code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -156,6 +156,14 @@ GLOBAL_LIST_EMPTY(TabletMessengers) // a list of all active messengers, similar
 		var/obj/item/computer_hardware/card_slot/card_slot = all_components[MC_CARD]
 		if(card_slot2?.try_eject(user) || card_slot?.try_eject(user))
 			return TRUE
+		
+		if(inserted_pai) // Remove pAI
+			user.put_in_hands(inserted_pai)
+			to_chat(user, span_notice("You remove [inserted_pai] from the [name]."))
```

Comment:
if(user.canUseTopic(src, BE_CLOSE))

Reply:
I think you missed that the pAI code is nested under the `canUseTopic` check for IDs.  See:

https://github.com/tgstation/tgstation/blob/0456b509c74ddb641ebbcaaaad65824226a5cbdc/code/modules/modular_computers/computers/item/computer.dm#L149-L165